lte/gateway/python/magma/pipelined/app/inout.py

```diff
@@ -335,7 +337,7 @@ def _setup_non_nat_monitoring(self, datapath):
                              self.config.non_nat_arp_egress_port,
                              self._uplink_port)
 
-        self._dhcp_gw_info = UplinkGatewayInfo(store.GatewayInfoMap())
+        self._dhcp_gw_info = UplinkGatewayInfo(store.GatewayInfoMap(self.redis_client))
```

Comment:
OBS: Had to inject redis_client here, since we removed its creation from `GatewayInfoMap`